ui/src/common/containers/__tests__/SearchBoxContainer.test.jsx

```diff
@@ -55,4 +55,67 @@ describe('SearchBoxContainer', () => {
     ];
     expect(store.getActions()).toEqual(expectedActions);
   });
+
+  it('resets the hash on search in collection different to literature', async () => {
+    const currentNamespace = 'literature';
+    const newNamespace = 'authors';
+    const store = getStoreWithState({
+      search: fromJS({
+        currentNamespace,
+      }),
+      router: {
+        location: {
+          hash: '#whatever',
+        },
+      },
+    });
+    const wrapper = mount(
+      <Provider store={store}>
+        <SearchBoxContainer />
+      </Provider>
+    );
+    const onSearch = wrapper.find(SearchBox).prop('onSearch');
+    onSearch(newNamespace, 'test');
+
+    const expectedAction = {
+      type: '@@router/CALL_HISTORY_METHOD',
+      payload: {
+        args: [{ hash: '' }],
+        method: 'replace',
+      },
+    };
+
+    expect(store.getActions()).toContainEqual(expectedAction);
+  });
+
+  it('does not reset the hash on search in literature', async () => {
```

Comment:
this is already tested in the first test case.

Reply:
Is it? The other one does the search in authors, this one does it in literature. The behaviour should be different.